src/token-list.json

```diff
@@ -1797,6 +1797,17 @@
       "coingeckoId": "darleygo-essence"
     }
   },
+  {
+    "chainId": 101,
+    "address": "CJiyMgq41YYH9McFQANw69KWyj74H8NjgVauaS1mfs39",
+    "symbol": "DEST",
+    "name": "Destiny",
+    "decimals": 9,
+    "logoURI": "https://raw.githubusercontent.com/Anonymous-Pentester/logo/main/logo1.png",
+    "extensions": {
+      "coingeckoId": "solana"
```

Comment:
This is incorrect and misleading and would lead to `DEST` having the price of SOL. Please fix the PR with the correct CG ID.